README.md

```diff
@@ -13,18 +19,39 @@ npm install cf-buttons
 npm install cf-expandables
 ```
 
+### Project scaffolding
+
 Want some boilerplate?
+There is a yeoman generator through the
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/351/commits/8293224f85be823317aa0844f8295a8803cc8b53